clusters/privatek8s.yaml

```diff
@@ -61,12 +60,16 @@ releases:
       - "../config/ext_jenkins-infra_privatek8s.yaml"
     secrets:
       - "../secrets/config/infra.ci.jenkins.io/jenkins-secrets.yaml"
-  - name: jenkins-infra-additional
-    namespace: jenkins-infra
-    chart: jenkins-infra/jenkins-additional
-    version: 0.0.3
```

Comment:
Good catch. The need for a service account with `cluster-admin` cluster-role (e.g. role applied cluster-wide) was removed in https://github.com/jenkins-infra/kubernetes-management/pull/1861/files#diff-f81821a2c2b64550a4ad4af0a390e2d173489247288e35b23720dceb759274d4 and I never cleaned up.


We'll have to uninstall the resources after merging this PR.

Can you remove the change from this PR though? It deserves another PR (that I'll make)